src/compiler.ts

```diff
@@ -7026,12 +7029,96 @@ export class Compiler extends DiagnosticEmitter {
     return module.unreachable();
   }
 
+  compileFirstClassFunction(
+    expression: FunctionExpression,
+    contextualType: Type,
+    constraints: Constraints
+  ): ExpressionRef {
+    let module = this.module;
+    let instance = this.doCompileFunctionExpression(expression, contextualType, constraints);
+    let currentType = this.currentType;
+    if (!instance) return module.unreachable(); 
+    let rtInstance = assert(this.resolver.resolveClass(this.program.functionPrototype, [ instance.type ]));
+    const functionIndexInTable = this.registerFunctionInTable(instance);
+    let ctor = this.ensureConstructor(rtInstance, expression);
+    let indexMember = assert(rtInstance.getMember("_index"));
+    assert(indexMember.kind == ElementKind.PropertyPrototype);
+    let indexProperty = assert((<PropertyPrototype>indexMember).instance);
+    let indexSetter = assert(indexProperty.setterInstance);
+    let tmp = this.currentFlow.getTempLocal(Type.i32);
+    let expr = module.block(null, [
+      this.makeCallDirect(
+        indexSetter,
+        [
+          module.local_tee(
+            tmp.index,
+            this.makeCallDirect(ctor, [module.i32(0)], expression, /*immediatelyDropped*/ false),
+            /*isManaged*/ true, // TODO(maybe can be false to optimize performance)
```

Comment:
```suggestion
            this.makeCallDirect(ctor, [module.i32(0)], expression, /* immediatelyDropped */ false),
            /* isManaged */ true, // TODO(maybe can be false to optimize performance)
```
This seems to trip up CI. Also, these sorts of comments generally aren't used in AS's source AFAIK, so maybe they should be removed later.